cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc

```diff
@@ -651,6 +651,18 @@ Tests for the absence of a pattern predicate if an expression predicate evaluate
 |
 |
 
+| <<query-plan-directed-union-relationship-types-scan,Directed Union Relationship Types Scan>>
+| Fetches all relationships and their start and end nodes with at least one of the provided types from the relationship type index.
+|
+|
+|
+
+| <<query-plan-undirected-union-relationship-types-scan,Undirected Union Relationship Types Scan>>
+| Fetches all relationships and their start and end nodes with at least one of the provided types from the relationship type index.
+|
+|
+|
+
```

Comment:
@pontusmelke I've added these operators to the ["at a glance" page](https://development.neo4j.dev/docs/cypher-manual/5.0/execution-plans/operator-summary/)

When this is building, I'll grab a screenshot for how it renders. If any entries are required in the `Leaf?`,  `Updating?` or `Condiderations` columns, please let me know.

Reply:
![Screenshot 2022-07-22 at 10 25 28](https://user-images.githubusercontent.com/12032446/180409065-e4bf736d-f3c9-4ced-a2f0-af04619b3e34.png)